weave/trace/custom_objs.py

```diff
@@ -52,31 +56,35 @@ def decode_custom_obj(
     encoded_path_contents: Mapping[str, Union[str, bytes]],
     load_instance_op_uri: Optional[str],
 ) -> Any:
-    load_instance_op = None
-    if load_instance_op_uri is not None:
+    _type = weave_type["type"]
+    found_serializer = False
+
+    # First, try to load the object using a known serializer
+    if _type in KNOWN_TYPES:
```

Comment:
this is still not quite right - we need to try with the local one first, else fallback to the the remote one

Reply:
sorry, asked for review too soon before pushing commit -- can you check again